torchaudio/datasets/vctk.py

```diff
@@ -11,11 +11,11 @@
     walk_files
 )
 
-URL = "http://homepages.inf.ed.ac.uk/jyamagis/release/VCTK-Corpus.tar.gz"
+URL = "https://datashare.is.ed.ac.uk/bitstream/handle/10283/3443/VCTK-Corpus-0.92.zip"
```

Comment:
From https://datashare.is.ed.ac.uk/handle/10283/3443, there is also a download link to http://datashare.is.ed.ac.uk/download/DS_10283_3443.zip . Do you know the difference?

I am downloading it now, but it takes a while.

Reply:
They are both the same, `http://datashare.is.ed.ac.uk/download/DS_10283_3443.zip` contains an extra License and readme files.